File: deps/src/cftmq/lua/receiver.lua
require('luastomp')

con=stomp.connect('127.0.0.1:40090')

if con then

    if con:login('root','') and con:subscribe('test') then

        while true do
            local msg=con:recv()

            if not msg then break end

            print(msg.destination,msg.data)

            con:ack()
        end

    end

end

File: deps/src/cftmq/test/receiver.cpp
#include "stompc.h"

int main(void)
{
    stomp::connection c;

    if(c.connect("127.0.0.1:40090"))
    {
        if(c.login("root","") && c.subscribe("test"))
        {
            stomp::frame f;
            
            while(c.recv(f))
            {
                printf("%s\n",f.data.c_str());

                if(!c.ack(f.hdrs["message-id"]))
                    break;
            }

            c.logout();
        }
    }

    return 0;
}
